index.js

```diff
@@ -16,10 +17,10 @@ const isVerboseMode = process.env.VERBOSE !== undefined;
 // eslint-disable-next-line
 const getQueryHash = (query) => isEmpty(query) ? '' : JSum.digest(query, 'SHA256', 'hex');
 
-const setRequestStatus = (res, code, message) => {
+const setRequestStatus = (res, code) => {
```

Comment:
cleaning up this method to not require you to tell it what a http status code is